proxy/http_proxy.go

```diff
@@ -84,6 +84,11 @@ func (p *HTTPProxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if t.AccessDeniedHTTP(r) {
+		http.Error(w, "access denied", http.StatusForbidden)
```

Comment:
I'm pretty sure that someone will ask for a different status code here but we can leave it for now.

Reply:
I can see that being a request but not sure there is much more appropraite than forbidden for a denied request.  Would you want that to be a config option at some point?

Reply:
yeah, if someone asks for this.